src/weather_magic/event_handlers.cljs

```diff
@@ -10,10 +11,25 @@
     (cam/perspective-camera
      (assoc camera-map :fov (min 140 (+ cur-val (* cur-val scroll-distance 5.0E-4)))))))
 
+(defn resize-handler [event]
```

Comment:
Eftersom vi inte länge använder event i den här funktionen, men en event handler måste per spec ta ett argument, så bör argumentet `event` döpas om till `_` per konvention: http://stackoverflow.com/questions/16020278/clojure-what-does-do-in-a-functions-argument-list